src/components/IdEntry.vue

```diff
@@ -56,7 +56,7 @@
     <span class="mx-auto text-red-700 text-base mb-1" v-if="isInvalidLoginMessageShown">{{
       invalidLoginMessage
     }}</span>
-    <!-- add srn button -->
+    <!-- add button -->
```

Comment:
please be a bit more descriptive with the comments so that it is immediately clear what something is doing. 

```suggestion
    <!-- button to add another input -->
```